### PR TITLE
Remove deprecated preview=markdown & adapt documentation

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
CC @wilzbach

### CODE REVIEW

File: spec/ddoc.dd [hunk 1]
```diff
@@ -513,7 +513,7 @@ $(P
 ------------------------------------
 
 $(P
-$(EM Enable support for backticks, tildes and other code languages with the `-preview=markdown` compiler flag.)
+$(EM Enable support for backticks, tildes and other code languages with the `-preview=markdown` compiler flag - enabled by default in 2.094.)
 )
```
Comment (wilzbach) [inline spec/ddoc.dd:517]: Why not simply remove this comment?

Comment (MoonlightSentinel) [reply spec/ddoc.dd:517]: Wasn't sure if the docs should cater towards older compilers

Comment (wilzbach) [reply spec/ddoc.dd:517]: No, they don't need to. We have doc archives for that.

Comment (MoonlightSentinel) [reply spec/ddoc.dd]: Done

**Review by wilzbach — APPROVED** [review]

**Pull request merged**